whereami/static/js/creategamemap.js

```diff
@@ -104,8 +104,47 @@ $(document).ready(function () {
       }
     });
 
+    var importInput = $('#importFileInput');
+    importInput.on('change', handleImportFile);
+
 });
 
+function handleImportFile() {
+    $('#infoText').text("");
+    files = $('#importFileInput').prop('files');
+    if (files.length === 0) {
+        console.log("no files selected");
+    } else {
+        for (file of files) {
+            var reader = new FileReader();
+            reader.readAsText(file, "UTF-8")
+            reader.onload = function (evt) {
+                try {
+                    var fileContent = evt.target.result;
+                    var obj = JSON.parse(fileContent);
+                    var mapName = $('#handpickedName');

```

Comment:
bei mehreren files wird der mapName hier vom letzten file genommen. Ich würde hier davon ausgehen, dass nur ein file selected sein darf. Bitte den for loop entfernen.

Reply:
Nein, es wird doch der erste genommen (wenn da einer vorhanden ist).
Die Loop ist aber tatsächlich nicht sinnvoll. Kann ich entfernen